configuration.adoc

```diff
@@ -1803,6 +1810,101 @@ Example, "userSMIMECertificate, UserCertificate"
 
 To set up another external LDAP server, click *Add Configuration*.
 
+=== Setting up server based S/MIME feature
+
+==== Prerequisites
+
+Same as the previous S/MIME version except that the Java is not required on client machine
+
+==== S/MIME License
+
+Same as the previous S/MIME version
+
+==== Enable S/MIME Feature
+
+Same as the previous S/MIME version
+
+==== Importing S/MIME Certificates
+
+Same as the previous S/MIME version except that the recipients' public-key certificate no longer need to be stored into Local OS or browser keystore.
+The certificate can be published to all other places mentioned in previous S/MIME version.
+
+==== List of LDAP attributes introduced in new S\MIME feature
```

Comment:
Suggest:
List of LDAP attributes introduced to support the server based S/MIME solution